core/src/Revolution/Processors/Workspace/Lexicon/ReloadFromBase.php

```diff
@@ -41,42 +42,54 @@ public function getLanguageTopics()
      */
     public function process()
     {
-        $language = 'en';
-        $namespace = 'core';
-        $corePath = $this->modx->getOption('core_path', null, MODX_CORE_PATH);
+        $namespace = $this->getProperty('namespace');
+        $topic = $this->getProperty('lexiconTopic');
+        $language = $this->getProperty('language');
+
+        $namespaceObj = $this->modx->getObject(modNamespace::class, $namespace);
+
+        if (empty($namespace) || empty($topic) || empty($language) || empty($namespaceObj)) {
+            $this->modx->log(modX::LOG_LEVEL_WARN, $this->modx->lexicon('lexicon_revert_error'));
+            $this->modx->log(modX::LOG_LEVEL_INFO, 'COMPLETED');
```

Comment:
I know you're following what was done later in this class, but I get the feeling this was not meant to make it into production code. If this logging does have some use, the content should point to a lexicon and not be hard-coded as “COMPLETED.” My guess is this log and the one later on should be removed.
```suggestion
            $this->modx->log(modX::LOG_LEVEL_INFO, 'COMPLETED');
```

Reply:
The `COMPLETED` log message tells the MODx.console that we're done

Reply:
While that may be the case, shouldn't it _not_ be hard-coded in English?

Reply:
well it only works in english, it could be a constant somewhere, but not a scope of this PR to change how it works

Reply:
A couple things, and I could be wrong on this (and in any case it's not a big deal at all), but...
- I believe this functionality works for all languages
- Looking at the entire file, I just don't see how this info-level log provides _any_ useful information

Again, it's nothing to haggle about, I just figure if there's an opportunity to clean up something in the process why not? ;-)

Reply:
@smg6511 — it is just the way the console works. It has to get a "COMPLETED" log message—exactly that string—to know the logging should be returned back away from the console (i.e. to whatever log settings are configured in the MODX instance).

Reply:
https://github.com/modxcms/revolution/blob/3.x/core/src/Revolution/Processors/System/Console.php#L86

Reply:
Thanks for that clarification! Now I get why John said it's out of scope to change how it works. (Ya gotta admit though, that's a pretty odd and un-intuitive way to communicate with the console component.)

Reply:
You will get no disagreement from us. 😉

Reply:
Quick question: Has marking a conversation as "resolved" changed recently? I'd like to mark this and my other comment on this subject, but see no way to do it now.

Reply:
I still see a "Resolve conversation" button at the end of each review comment thread. Are you not seeing this?

Reply:
I am not.